src/components/navigation-header/components/nav-bar-list-container/index.tsx

```diff
@@ -22,8 +23,8 @@ export default function NavBarListContainer({
 	children: ReactNode
 }) {
 	return (
-		<nav className={s.nav}>
+		<NavigationMenu.Root className={s.nav}>
 			<ul className={s.ul}>{children}</ul>
```

Comment:
If the nav is replaced by `<NavigationMenu.Root>`, should the `<ul>` also be replaced by `<NavigationMenu.List>`?

Reply:
That's a good idea @heatlikeheatwave, implemented in this commit:
0a2b19f